deploy/v3_1.js

```diff
@@ -144,6 +144,14 @@ module.exports = async ({ deployments, getChainId }) => {
     ],
   });
 
+  // Deploy a JBMigrationOperator contract.
```

Comment:
I would put it in a standalone deployment script (no need to deploy it with the whole protocol stuff), using another tags

Reply:
add in a separate sol script